audb/core/load.py

```diff
@@ -825,84 +888,110 @@ def load(
         sampling_rate=sampling_rate,
     )
     db_root = database_cache_folder(name, version, cache_root, flavor)
+    db_lock_path = database_lock_path(db_root)
+    db = None
 
-    if verbose:  # pragma: no cover
-        print(f'Get:   {name} v{version}')
-        print(f'Cache: {db_root}')
+    try:
+        with filelock.FileLock(db_lock_path, timeout=timeout):
```

Comment:
Great, seems like the `filelock` module does basically the job for us.

Reply:
Yes, there are other libraries, but this looked like the most common one. And the latest release is from February, so it's also actively developed.